src/shared/components/input/TextFieldWithLabelFormGroup.tsx

```diff
@@ -14,11 +14,12 @@ interface Props {
 
 const TextFieldWithLabelFormGroup = (props: Props) => {
   const { value, label, name, isEditable, isInvalid, isRequired, feedback, onChange } = props
-  const id = `${name}TextField`
+  const inputId = `${name}TextField`
   return (
     <div className="form-group">
-      {label && <Label text={label} htmlFor={id} isRequired={isRequired} />}
+      {label && <Label text={label} htmlFor={inputId} isRequired={isRequired} />}
       <TextField
+        id={inputId}
```

Comment:
```suggestion
        id={inputId}
        data-testid={inputId}
```
I am actually unsure about this one 😅 . Don't know if `id` will break on this library component as well.